atom/browser/api/atom_api_top_level_window.cc

```diff
@@ -262,6 +262,10 @@ void TopLevelWindow::OnWindowLeaveHtmlFullScreen() {
   Emit("leave-html-full-screen");
 }
 
+void TopLevelWindow::OnWindowAlwaysOnTopChange() {
+  Emit("always-on-top-change", IsAlwaysOnTop());
```

Comment:
s/change/changed for consistency with `accessibility-support-changed`, `accent-color-changed`, `cursor-changed`, etc.